reports.js

```diff
@@ -60,7 +29,10 @@ async function pluginsReport() {
     const url = plugin.url || '';
     plugin.name = key;
     plugin.installs = plugin.installs || 0;
-    if (url.match('https?://github.com/jenkinsci/')) {
+    if (isTombstoned[key]) {
```

Comment:
Could this be the last branch rather than first, so that deprecated plugins that use GH for readme still show as OK? Also Deprecated would be imho better than TOMBSTONED and className could be `dark`.

Reply:
oh nice, i was debating if we wanted a different classname

and yea, absolutely on the first note, good idea, I moved it when i was debugging and forgot to move it back.